src/axom/lumberjack/MPIUtility.cpp

```diff
@@ -49,16 +49,49 @@ const char* mpiBlockingReceiveMessages(MPI_Comm comm)
   return charArray;
 }
 
+const char* mpiNonBlockingReceiveMessages(MPI_Comm comm, const int tag)
+{
+  const int mpiTag = (tag == false) ? LJ_TAG : tag;
```

Comment:
I think this should be changed to compare `tag` to an integer value not a bool. Also, the `const` qualifier is not needed on the tag argument because it is passed by value.